.travis/install.sh

```diff
@@ -2,16 +2,15 @@
 
 [ -n "$CGAL_DEBUG_TRAVIS" ] && set -x
 DONE=0
-sudo add-apt-repository ppa:mhier/libboost-latest
+sudo add-apt-repository ppa:mhier/libboost-latest -y
 sudo apt-get update
 
 while [ $DONE = 0 ]
 do
   DONE=1 && sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install clang zsh \
 flex bison cmake graphviz libgmp-dev libmpfr-dev libmpfi-dev zlib1g-dev libeigen3-dev  \
 qtbase5-dev libqt5sql5-sqlite libqt5opengl5-dev qtscript5-dev libqt5svg5-dev qttools5-dev qttools5-dev-tools qml-module-qtgraphicaleffects libopencv-dev mesa-common-dev libmetis-dev libglu1-mesa-dev \
-libboost
-|| DONE=0 && sudo apt-get update
+libboost || DONE=0 && sudo apt-get update
```

Comment:
The name of the package, in the PPA is `libboost1.73-dev`:
https://launchpad.net/~mhier/+archive/ubuntu/libboost-latest/+packages?field.name_filter=&field.status_filter=published&field.series_filter=xenial

For the moment, the Travis tests fail with an infinite loop:

https://travis-ci.com/github/CGAL/cgal/jobs/366968403
https://travis-ci.com/github/CGAL/cgal/jobs/366968447